akka-docs/src/test/scala/docs/stream/operators/sink/Fold.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sink
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.{ Sink, Source }
+
+import scala.concurrent.Future
+
+object Fold {
+  implicit val system: ActorSystem = ???
+
+  def foldExample: Future[Int] = {
+    //#fold
+    Source(1 to 100).runWith(Sink.fold(0)(_ + _))
+    //Future(Success(5050))
```

Comment:
Same as for Java, given that this will be read by users that are potentially newcomers to even async programming it is good with complete act-on-materialized-value logic.